src/charts/topology/examples/topology-view.js

```diff
@@ -0,0 +1,309 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.component:pfTopology
+ * @restrict E
+ *
+ * @description
+ *   Component for rendering a topology chart.  Individual nodes and relationships can be represented with this view.  CSS is especially important for rendering the noes and lines.  The example inline contains specific examples that can be used to change the icon size and the line type of the relationships.
+ *
+ *   In addition; searching, filtering and label visibility is also supported.<br/>
+ *
+ * @param {object} items items to display in the topology chart, each is represented as an individual node.  The keys of this object are used in the relations attribute. The items should have a item.kind attribute, as well as the usual item.metadata and so on.:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.name    - name of the item the node represents
+ * <li>.status  - optional status of the node (can be used to differentiate the circle color)
+ * <li>.kind    - the kind of node
```

Comment:
Can you elaborate on what 'kinds' of charts are supported, and what the 'kind' values may be?
Thanks,
- Dave